recipes/pyflyby/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyflyby" %}
+{% set version = "1.6.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5bebd9fe0f40d809bbf029e3d0d1bfa7bfe2a94379f5a8f383cb85d94fb9a88c
+
+build:
+  skip: true  # [win]
+  number: 0
+  entry_points:
+    - py = pyflyby._py:py_main
+    - py3 = pyflyby._py:py_main
```

Comment:
These do not seem like the correct entry points.  py and py3 are for python itself, I thought

Reply:
These are what pyflyby installs.

Reply:
Hmmm They shouldn't. But OK